query-engine/driver-adapters/js/connector-test-kit-executor/src/index.ts

```diff
@@ -170,10 +181,39 @@ function respondOk(requestId: number, payload: unknown) {
 }
 
 async function initQe(url: string, prismaSchema: string, logCallback: qe.QueryLogCallback): Promise<[engines.QueryEngineInstance, ErrorCapturingDriverAdapter]> {
-    const pool = new pgDriver.Pool({ connectionString: url })
-    const adapter = bindAdapter(new pg.PrismaPg(pool))
-    const engineInstance = qe.initQueryEngine(adapter, prismaSchema, logCallback)
-    return [engineInstance, adapter];
+    const adapter = await adapterFromEnv(url) as DriverAdapter
```

Comment:
```suggestion
    const adapter = await adapterFromEnv(url)
```
this shouldn't be necessary here since `adapterFromEnv` already returns a `Promise<DriverAdapter>`

Reply:
Yup, left over from dynamic instantiation through `globals` Consider it addressed.